docs/manage/fleet/robots.md

```diff
@@ -56,12 +52,23 @@ If you are managing a large fleet, you can use {{< glossary_tooltip term_id="fra
 
 ### Logs
 
-Each robot automatically sends logs to the cloud which you can view on the **LOGS** tab.
+To make debugging issues with your robots easier, each robot automatically sends its logs to the cloud.
+You can access your logs from the **LOGS** tab in the [Viam app](https://app.viam.com) and filter your logs for specific keywords or log levels:
+
+{{<gif webm_src="../../img/log-filtering.webm" mp4_src="../../img/log-filtering.mp4" alt="Filter logs by term of log level in the UI">}}
+
+You can also change your timestamp format to ISO or Local depending on your preference.
 
 ### History
 
 The configuration of your robot and the code it runs are kept separate to make debugging easier.
-The **HISTORY** tab shows a timestamped diff view of your robot's configuration changes.
+The **HISTORY** tab shows timestamped changes to your robot's configuration.
+
+If you want to revert changes that you made, you can load a previous configuration by clicking the **LOAD CONFIG** button next to the respective configuration.
+
+{{<gif webm_src="../../img/load-prev-config.webm" mp4_src="../../img/load-prev-config.mp4" alt="Load a previous config from the UI">}}
```

Comment:
Unfortunately this gif looks super blurry when 
I load it on the staging link-- maybe try reuploading?